### ISSUE TITLE
how can i close an app after i opened it?-

### ISSUE THREAD

**Issue by RoyVidav** [robotemi/sdk#328]
when i use my temi (spotify for example) and i set temi on standby mode, all the apps are still open in the back.
this issue make the temi run really slow beacuse of the over flow i think.
how can i close it?

**Comment by zjn0505** [comment]
There isn't a way to manage background apps, unless you manually install a navigation bar app on the robot.

**Comment by RoyVidav** [comment]
Hey!
thank you for your answer!

do you have this code to send me?
if its unfinished mybe i can work on it also.

roy

בתאריך יום ד׳, 9 בנוב׳ 2022, 12:56, מאת zjn0505 ***@***.***>:

**Comment by zjn0505** [comment]
No, I mean install an app like this one. Then you can show the recent tasks and force close one.
https://apkpure.com/navigation-bar-for-android/nu.nav.bar/download/18-APK?from=versions%2Fversion

On V2 robot, you can also request accessibility permission and trigger recent task by `AccessibilityService.performGlobalAction(AccessibilityService.GLOBAL_ACTION_RECENTS)`

**Comment by RoyVidav** [comment]
Thanks, I'll check it out.

בתאריך יום ה׳, 10 בנוב׳ 2022, 07:13, מאת zjn0505 ***@***.***
